server/src/handlers/helpers.ts

```diff
@@ -0,0 +1,29 @@
+import { Connection } from 'vscode-languageserver';
+
+export function isCloudFormation(template: string, filename: string, cfnConnection: Connection): Boolean {
+
+	if (/"?AWSTemplateFormatVersion"?\s*/.exec(template)) {
+		cfnConnection.console.log("Determined this file is a CloudFormation Template. " + filename +
+			". Found the string AWSTemplateFormatVersion");
+		return true;
+	}
+	if (/\n?"?Resources"?\s*:/.exec(template)) {
+		if (/"?Type"?\s*:\s*"?'?[a-zA-Z0-9]{2,64}::[a-zA-Z0-9]{2,64}/.exec(template)) {
+			// filter out serverless.io templates
+			if (!(/\nresources:/.exec(template) && /\nprovider:/.exec(template))) {
```

Comment:
I had a small issue that I had commented out the resources section and cfn-lint determined that it's a cloudformation file, perhaps would be better to check `functions:` & `provider:`?